py3status/modules/wifi.py

```diff
@@ -136,11 +132,20 @@ def _dbm_to_percent(self, dbm):
     def _percent_to_dbm(self, percent):
         return (percent / 2.0) - 100
 
+    def _get_wifi_data(self, command):
+        for time in range(2):
+            try:
+                return self.py3.command_output(command)
+            except self.py3.CommandError as ce:
+                if str(command) not in self.storage:
```

Comment:
can we replace `self.storage` with `if commmand[0] != 'sudo'`?

Reply:
That's not what it is doing here. If you manage to run fine on startup without `sudo` and it later throw an error, we use this to determine that we already ran this command before as-is, show error instead of adding `sudo` and try again.

Reply:
I see, can we rename `storage` to something that explains the purpose of this variable? `attempted_commands`?

Reply:
Successful commands goes in `commands`. Failed commands gets sudo then error. Successful commands already in `commands` gets an error (should be rare). Hope that's better.

Reply:
Did you mean to rename `self.storage` to `self.commands`?

Reply:
Done. Yes. I didn't notice this branch not on upstream. Should we merge PRs or wait for others?

Reply:
I pinged ultrabug on IRC earlier today, let's give him another day or two 😉